modules/openxBidAdapter.js

```diff
@@ -187,10 +203,25 @@ const OpenxAdapter = function OpenxAdapter() {
       }
     });
 
-    params.callback = 'window.$$PREBID_GLOBAL$$.oxARJResponse';
-    let queryString = buildQueryStringFromParams(params);
+    if (window.XMLHttpRequest && 'withCredentials' in new XMLHttpRequest()) {
```

Comment:
Unfortunately we require the credentials for our ajax support to function correctly, so we must do this check here